corehq/apps/linked_domain/applications.py

```diff
@@ -118,6 +118,10 @@ def link_app(linked_app, master_domain, master_id, remote_details=None):
     return linked_app
 
 
+def link_app_in_existing_domain_link(domain_link, original_app_id):
```

Comment:
I think the presence of the `domain_link` arg implies there's an existing domain link.

Reply:
(resolved in later commit)